lib/Cake/Model/Datasource/CakeSession.php

```diff
@@ -468,14 +468,19 @@ public static function destroy() {
 	}
 
 /**
- * Clears the session, the session id, and renews the session.
+ * Clears the session.
  *
+ * Optionally also clears the session id and renews the session.
+ *
+ * @param bool $renew If the session should also be renewed. Defaults to false.
  * @return void
  */
-	public static function clear() {
+	public static function clear($renew = false) {
 		$_SESSION = null;
-		self::$id = null;
-		self::renew();
```

Comment:
This is an interesting behavior change that may surprise people. The old default was to renew the session, but now its not.